lib/travis/api/v3/models/cron.rb

```diff
@@ -82,7 +82,7 @@ def always_run?
 
     def last_non_cron_build_time
       last_build = Build.find_by_id(branch.last_build_id)
-      last_build.finished_at.to_datetime.utc if last_build && last_build.finished_at
+      last_build.started_at.to_datetime.utc if last_build && last_build.started_at
```

Comment:
could use the ruby 2.3 `&.` safe navigation operator (http://nithinbekal.com/posts/ruby-2-3-features/).